.github/workflows/release.yml

```
@@ -102,6 +102,7 @@ jobs:
         with:
           name: ${{ matrix.artifact_name }}
           path: ${{ matrix.binary_path }}
+          flatten: true
```

actionlint (1.7.11)

[error] input "flatten" is not defined in action "actions/upload-artifact@v4". available inputs are "compression-level", "if-no-files-found", "include-hidden-files", "name", "overwrite", "path", "retention-days"

(action)